src/commands/generate/__tests__/cliTest.ts

```diff
@@ -186,8 +186,8 @@ describe("generate command", () => {
         expect(fs.existsSync(path.join(outDir, "node_modules/typescript/bin/tsc")));
     });
 
+    jest.setTimeout(10000);
```

Comment:
why do we move this out of the test case? seems like other test cases still set the timeout explicitly

Reply:
I had a similar issue as https://github.com/facebook/jest/issues/11543 it's simply ignored, I should probably spend more time looking into it

Reply:
lifted this up to the top level describe and remove from each individual test, are you ok with that?